xml/ceph_maintenance_updates.xml

```diff
@@ -33,6 +33,258 @@
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Whether pools are replicated or erasure coded, and which erasure code
+      profile plug-in and parameters are in use.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      How many hosts are in the cluster, and how many hosts have each type of
+      daemon.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Whether a separate OSD cluster network is being used.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      How many RBD pools and images are in the cluster, and how many pools have
+      RBD mirroring enabled.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      How many RGW daemons, zones, and zonegroups are present and which RGW
+      frontends are in use.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Aggregate stats about the &crushmap;, such as which algorithms are used,
+      how big buckets are, how many rules are defined, and what tunables are in
+      use.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    If you had telemetry enabled before 14.2.5, you will need to re-opt-in
+    with:
+   </para>
+<screen>&prompt.cephuser;ceph telemetry on</screen>
+   <para>
+    If you are not comfortable sharing device metrics, you can disable that
+    channel first before re-opting-in:
+   </para>
+<screen>
+&prompt.cephuser;ceph config set mgr mgr/telemetry/channel_device false
+&prompt.cephuser;ceph telemetry on
+</screen>
+   <para>
+    You can view exactly what information will be reported first with:
+   </para>
+<screen>
+&prompt.cephuser;ceph telemetry show        # see everything
+&prompt.cephuser;ceph telemetry show device # just the device info
+&prompt.cephuser;ceph telemetry show basic  # basic cluster info
+</screen>
+  </listitem>
+  <listitem>
+   <para>
+    <emphasis role="bold">New OSD daemon command
+    <command>dump_recovery_reservations</command></emphasis>. It reveals the
+    recovery locks held (<option>in_progress</option>) and waiting in priority
+    queues. Usage:
+   </para>
+<screen>
+&prompt.cephuser;ceph daemon osd.<replaceable>ID</replaceable> dump_recovery_reservations
+</screen>
+  </listitem>
+  <listitem>
+   <para>
+    <emphasis role="bold">New OSD daemon command
+    <command>dump_scrub_reservations</command>. </emphasis> It reveals the
+    scrub reservations that are held for local (primary) and remote (replica)
+    PGs. Usage:
+   </para>
+<screen>
+&prompt.cephuser;ceph daemon osd.<replaceable>ID</replaceable> dump_scrub_reservations
+</screen>
+  </listitem>
+  <listitem>
+   <para>
+    <emphasis role="bold">RGW now supports S3 Object Lock set of
+    APIs.</emphasis> RGW now supports S3 Object Lock set of APIs allowing for a
+    WORM model for storing objects. 6 new APIs have been added PUT/GET bucket
+    object lock, put/get object retention, put/get object legal hold.
```

Comment:
There's also lowercase put/get here too.
Should it be <literal>PUT/GET</literal> for all three?

Reply:
ack, uppercase these